pages/national_resources/cz_resources.md

```diff
@@ -0,0 +1,107 @@
+---
+
+title: Czech Republic
+country_code: CZ
+contributors: [Karel Berka]
+coordinators: [Karel Berka]
+
+# Link to other pages in the tool assembly section on the RDMkit by listing the page_id 
+related_pages:
+  tool_assembly: [ ]
+
+training:
+  - name: Training in TeSS
+    registry: TeSS
+    url: https://tess.elixir-europe.org/content_providers/elixir-the-czech-republic-node
+  - name: ELIXIR CZ YouTube
+    registry: Youtube
+    url: https://www.youtube.com/channel/UCt0SKet24szBGjN-V1d4EKw
+  - name: ELIXIR-CZ Github
+    registry: Github
+    url: https://github.com/ELIXIR-CZ
+ - name: ELIXIR CZ Zenodo
+    registry: Zenodo
+    url: https://zenodo.org/communities/elixir-cz/?page=1&size=20
+
+ref_to_main_resources: 
+  -  Galaxy
+
+national_resources: 
+  - name: Galaxy Metacentrum
+    description: Galaxy Metacentrum is a Galaxy instance managed by the Czech ELIXIR node and e-Infra. It provides extra support for [RepeatExplorer](https://repeatexplorer-elixir.cerit-sc.cz/) tool for plant genomic analysis
+    how_to_access: 
+    instance_of: Galaxy
+    related_pages:
+      tool_assembly: [ ]
+      your_domain: [plant_sciences]
```

Comment:
hello, make sure that the page ids are the ones listed here https://rdmkit.elixir-europe.org/website_overview (without underscores)

Reply:
Well it is one of the sustainability topics I would love to talk about on our Hackathon :)

Reply:
Can I help here with something? I am a bit puzzled what I should do next to help the page be up and running...